lms/djangoapps/teams/static/teams/js/collections/base.js

```diff
@@ -27,7 +27,7 @@
                     var self = this,
                         deferred = $.Deferred();
                     if (force || this.isStale) {
-                        this.fetch()
+                        this.setPage(1)
```

Comment:
Thanks for the explanation. Still, I'm wondering why we always reset to the first page. Why not call `setPage(this.currentPage)`?